test/__init__.py

```diff
@@ -59,26 +59,25 @@ def run_role_meta_main(self, meta_main_text):
 
 
 def run_ansible_lint(
+        *argv: str,
         cwd: os.PathLike = None,
-        role_path: os.PathLike = None,
         bin: str = None,
-        env: Dict[str, str] = None):
+        env: Dict[str, str] = None) -> Tuple[str, str]:
     """Run ansible-lint on a given path and returns its output."""
     if bin:
-        command = [bin, "-v"]
+        command = [bin]
     else:
-        command = [sys.executable, "-m", "ansiblelint", "-v"]
```

Comment:
I think removing `-v` will affect all the tests relying on verbosity being set...

Reply:
It does not affect because we did not check stderr anyway. Lucky this time.